discord/ext/commands/core.py

```diff
@@ -43,8 +43,10 @@
 )
 
 import discord
+import discord.utils.private
+from ...utils.private import evaluate_annotation
```

Comment:
```suggestion
from ...utils.private import evaluate_annotation, maybe_awaitable, async_all
```
and then fix in the 2 places below